PukingPredator/Assets/Scripts/ItemReplace.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+
+public class ItemReplace : MonoBehaviour
+{
+    public GameObject nextPrefab;
+
+    /// <summary>
+    /// Returns the prefab that will replace the current one
+    /// </summary>
+    public GameObject GetNext()
```

Comment:
can we access nextPrefab directly instead of using GetNext, and possibly set the setter to private if we don't need it?

Reply:
yes ill make is a private serialized Field

Reply:
If it's replacing the public function, it can't be a private serialized field. the swap is just removing GetNext and changing it from:
`
[SerializeField]
private GameObject nextPrefab;`
to:
`public GameObject nextPrefab { get; private set; }`
(serialize field also needs to go, its intended to make something private appear in the unity inspector, but it will be public after the change)